### DISCUSSION TITLE
How does Dawarich handle duplicate imports

### DISCUSSION THREAD

**Discussion by TheBig-O** [Freika/dawarich#306] [Q&A] [answered]
I'm excited to try the functionality you added with the "Watch" folder. I have a question about how imports are handled though. If for some reason there are duplicate days or overlaps between imported files, does Dawarich disregard those points or will it double up? (I'm trying to consolidate both Nextcloud and Google into Dawarich.) 
I'm not sure it matters, but did want to know what to expect before attempting to import a file from Google's timeline and Nextcloud.
Thanks for the great app and the information.

**Comment by Freika** [comment, marked as answer]
Point uniqueness is being checked before creation based on latitude, longitude, timestamp and user id. If in the database there is already a point with the same attributes as the one that is about to be created, no duplicate will be added and importing process will simply move on to the next point from the imported file.

**Reply by TheBig-O** [reply]
Excellent! That was what I was hoping for. This makes getting a complete picture of past travels much easier.
THANKS!

**Comment by eng3** [comment]
Does the duplicate need to be exactly the same with all attributes?  What if my logger contributes a lat/lon and a photo contributes a lat/lon that is "nearly" identical but off by a few meters? or if the times are off by a few seconds/minutes, etc.

**Reply by Freika** [reply]
Deduplication is based on same coordinates and timestamp. If two points have them slightly different, both will be created
